src/lib/render-error-remplate.js

```diff
@@ -0,0 +1,17 @@
+const { readFile } = require('fs').promises
+const { join } = require('path')
+
+let errorTemplateFile
+
+const renderErrorTemplate = async (errString, templatePath = './templates/function-error.html') => {
```

Comment:
It feels a bit weird to have this default to the template for serverless functions. I think we shouldn't have a default, and then the consumer always has to specify the path for the template.

This can totally be done in a follow up, though.

Reply:
Nah that's an easy change! Will do!